docs/update-locking.md

```diff
@@ -9,6 +9,8 @@ Locking updates means that the resin.io device supervisor will not be able to ki
 
 In order to do this, users can create a lockfile called `resin-updates.lock` in a way that it has exclusive access, which will prevent the device supervisor from killing and restarting the app. As with any other lockfile, the supervisor itself will create such a file before killing the app, so you should only create it in exclusive mode. This means that the lockfile should only be created if it doesn't already exist. The exclusive access is achieved by opening the lockfile with the [O_EXCL and O_CREAT flags](https://linux.die.net/man/3/open), and several tools exist to simplify this process with examples given [below](#creating-the-lockfile).
 
+The presence of a lockfile will ensure that your application does not get killed, but updates will still be downloaded by the supervisor, ready to be applied once the lock has no longer been taken.
```

Comment:
"once the lock has no longer been taken" is a little awkward as the language in the paragraph above doesn't mention taking the lock, it mentions creating and opening the lockfile. Is there a way to state this that is more consistent with the description above?